lang/cs/Org.Apache.REEF.Driver/Bridge/Events/EvaluatorRequestor.cs

```diff
@@ -63,13 +67,24 @@ internal EvaluatorRequestor(IEvaluatorRequestorClr2Java clr2Java)
 
         public void Submit(IEvaluatorRequest request)
         {
-            LOGGER.Log(Level.Info, "Submitting request for {0} evaluators and {1} MB memory and  {2} core to rack {3}.", request.Number, request.MemoryMegaBytes, request.VirtualCore, request.Rack);
+            LOGGER.Log(Level.Info, "Submitting request for {0} evaluators and {1} MB memory and  {2} core to rack {3} and runtime {4}.", request.Number, request.MemoryMegaBytes, request.VirtualCore, request.Rack, request.RuntimeName);
 
+            byte[] data = Clr2Java.GetDefinedRuntimes();
+            DefinedRuntimes runtimes = DefinedRuntimesSerializer.FromBytes(data);
+            LOGGER.Log(Level.Info, "Defined runtimes " + ((runtimes.runtimeNames == null) ? "null" : string.Join(",", runtimes.runtimeNames)));
```

Comment:
Please move this into a different log level. It isn't needed most of the time, right?